pages.de/android/logcat.md

```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Gib ein Protokoll aller Systemmeldungen aus:
+> Weitere Informationen: <https://developer.android.com/studio/command-line/logcat>.
+
+- Gib ein Protokoll aller Systemmeldungen aus:
+
+`logcat`
+
+- Schreibe alle Systemmeldungen in eine Datei:
+
+`logcat -f {{pfad/zu/datei}}`
+
+- Gib Zeilen aus, die einem Regex entsprechen:
```

Comment:
```suggestion
- Gib Zeilen aus, die einem regulären Ausdruck entsprechen:
```
Not necessarily necessary… Just a suggestion.

Reply:
I also haven't looked into how other German pages do this, so 🤷🏻‍♀️

Reply:
I'm not entirely sure. In the english page we also use `regex` and not `regular expression`.

Reply:
I like the full name rather than the shortened jargon.  It sounds more professional.  Let's use it instead here and in the English page.

Reply:
You could also open a PR to standardize that.

Reply:
[`pages/common/httpflow.md`](https://github.com/tldr-pages/tldr/blob/521c8cd748736a1ef1da44e3650ff1e2dc30b333/pages/common/httpflow.md) uses `regexp`; that should be fixed too.

Reply:
Many pages use `{{regex}}` in their arguments, I assume that's fine. However, some use `regex` and some `regexp`. Should we normalize that as well?

Reply:
Yeah I'd say normalize it all to `{{regular_expression}}`.  It it's too long in some places we can nip those occurences during the review.